Rakefile

```diff
@@ -1,30 +1,20 @@
 # frozen_string_literal: true
 
-require 'cucumber/rake/task'
-require 'rspec/core/rake_task'
-require 'rubocop/rake_task'
-
-require 'yard'
-
-namespace :cuke do
-  Cucumber::Rake::Task.new(:all) do |t|
-    t.cucumber_opts = '--format pretty'
-  end
+task :cukes do
+  @cuke_exit_code = system 'bundle exec cucumber' || raise('Cukes failed')
```

Comment:
Not sure we need to store the exit code in a var, do we? The `|| raise ('Cukes failed')` fulfills the purpose of stopping the build upon failure, right?